From 410b579494a72c324ef054559bf330344c757df8 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Wed, 10 Sep 2014 18:28:48 -0400
Subject: [ethernet] ixgbe: fix some multiline hw_dbg prints

Message-id: <1410373750-11341-29-git-send-email-jogreene@redhat.com>
Patchwork-id: 93532
O-Subject: [RHEL7.1 PATCH 28/50] ixgbe: fix some multiline hw_dbg prints
Bugzilla: 1091123
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

BZ: 1091123
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

This patch fixes some formatting on multilined print messages, so that
the text of the print appears on a single line, which aids in grepping
the sourcecode for where the error came from.

Signed-off-by: Jacob Keller <jacob.e.keller@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 305f8cec7be51e5bf2074e10416133546afa117e)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbe/ixgbe_82599.c b/drivers/net/ethernet/intel/ixgbe/ixgbe_82599.c
index d3e8b8b..d638564 100644
--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_82599.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_82599.c
@@ -512,7 +512,7 @@ out:
  *
  * Disables link, should be called during D3 power down sequence.
  *
- */
+ **/
 static void ixgbe_stop_mac_link_on_d3_82599(struct ixgbe_hw *hw)
 {
  u32 autoc2_reg;
@@ -1005,8 +1005,7 @@ static s32 ixgbe_setup_mac_link_smartspeed(struct ixgbe_hw *hw,
 
 out:
  if (link_up && (link_speed == IXGBE_LINK_SPEED_1GB_FULL))
-  hw_dbg(hw, "Smartspeed has downgraded the link speed from "
-         "the maximum advertised\n");
+  hw_dbg(hw, "Smartspeed has downgraded the link speed from the maximum advertised\n");
  return status;
 }
 
@@ -1114,8 +1113,7 @@ static s32 ixgbe_setup_mac_link_82599(struct ixgbe_hw *hw,
     if (!(links_reg & IXGBE_LINKS_KX_AN_COMP)) {
      status =
              IXGBE_ERR_AUTONEG_NOT_COMPLETE;
-     hw_dbg(hw, "Autoneg did not "
-            "complete.\n");
+     hw_dbg(hw, "Autoneg did not complete.\n");
     }
    }
   }
-- 
1.7.1